### ISSUE TITLE
Relocate "Update Plugins" Execution Button

### ISSUE THREAD

**Issue by rokville** [swar8080/obsidian-plugin-update-tracker#45]
This really should be a Core plugin, great work here!  

Suggestion: Consider moving (or offering the option to move) the "Update Plugins" (the pop up button that comes after selecting plugins) to the top (or top-left) since the Selection toggles are on the left and the "Select All" toggle is on the top-left.  This would preclude the need to move the mouse from top-left to bottom-center to execute the request to update.

**Comment by Intoxinator** [comment]
Came here to add exactly this.

**Comment by SublimePeace** [comment]
Agreed, but not from a convenience standpoint but a usability and functionality standpoint:

there's also the issue of the update plugins popup at the bottom covering the changelog of the bottommost plugin. I didn't screenshot it, but just now I installed a bunch of long overdue updates, and I wasn't able to read the changelog for the last entry due to that floating "update plugins" rectangle.

I know this is a ~4 year old issue, but wanted to point this out regardless, in case development of this repo continues.
